### DISCUSSION TITLE
Dynamically task and task group creating based on Dag run configuration

### DISCUSSION THREAD

**Discussion by GhostNA** [apache/airflow#21521] [Q&A]
Hello. 
Is there a way to get Dag run configuration during task creating?

Idea is that divide one long process into parts (small tasks) which will handle Celery workers. 

How can I do it when DAG was triggered through API with a specific dag run configuration?

**Comment by potiuk** [comment]
You can't (Airflow 2.2). But there is a work in progress to allow that that is likely coming in Airflow 2.3.0

https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-42+Dynamic+Task+Mapping

**Comment by sanjana2015** [comment]
Is this feature available in the version of airflow 2.3.2 ??

**Reply by potiuk** [reply]
Yes.

**Reply by potiuk** [reply]
But I recommend to install to latest released 2.3* version
